src/sentry/tsdb/snuba.py

```diff
@@ -166,3 +166,38 @@ def flatten_keys(self, items):
             return (items.keys(), list(set.union(*(set(v) for v in items.values()))))
         else:
             return (None, None)
+
+    # The following are no-ops, but implemented here to avoid NotImplementedErrors everywhere.
```

Comment:
We would still raise `Exception: Unsupported TSDBModel:` in the case where a caller tries to do a read operation on a TSDBModel that SnubaTSDB doesn't understand.